### PR TITLE
Correct indefinite article a/an

### PR DESCRIPTION
**Pull request by eksperimental**
Run the following command, and reviewed the occurences one by one.
```sh
 # find words starting with vowels
ag -i '\ban?\s+[aeiou]\w+\b'

 # find acronyms
ag '\ban?\s+[BCDFGHJKLMNPQRSTVWXYZ][A-Z]+\b'

 # starting with "h"
ag -i '\ban?\s+[aeiou]\w+\b'
```

### CODE REVIEW

File: lib/elixir/lib/io.ex [hunk 1]
```diff
@@ -250,7 +250,7 @@ defmodule IO do
   Reads a line from the IO device. It returns:
 
     * `data` - the characters in the line terminated
-      by a LF (or end of file)
+      by an LF (or end of file)
```
Comment (DavsX) [inline lib/elixir/lib/io.ex]: I think this should stay 'a', not 'an' :)

Comment (eksperimental) [inline lib/elixir/lib/io.ex]: I can see your point. It all depends on how you read it.
I read it, "the characters in the line terminated by an el-ef"

An interesting util is http://home.nerbonne.org/A-vs-An/
which scanned Wikipedia in English and created a dictionary with matches like these ones.
97 times is used, vs 17 in favor or "an LF"

but I can change it to "a", any more opinions?

Comment (josevalim) [inline lib/elixir/lib/io.ex]: let's just replace LF by "line feed (LF)".

Comment (eksperimental) [inline lib/elixir/lib/io.ex]: I changed it a bit, to be more consistent.

    * `data` - the characters in the line terminated
      by a line-feed (LF) or end of file (EOF)

Comment (josevalim) [inline lib/elixir/lib/io.ex]: Beautiful. Let me know when you push it and I will merge this!

Comment (eksperimental) [inline lib/elixir/lib/io.ex]: done! .... sorry, I thought they were pushed

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**